src/components/LoanFinding/LendingCategorySection.vue

```diff
@@ -40,6 +40,7 @@
 						@add-to-basket="addToBasket"
 						@show-cart-modal="showCartModal"
 						@show-loan-details="showLoanDetails"
+						@mouseenter="$emit('mouse-enter-loan-card', loan?.id)"
```

Comment:
Looks like some changes from the other PR. Once this PR gets updated with that merged PR, I'll take another look.